### ISSUE TITLE
replace all occurrences not work

### ISSUE THREAD

**Issue by Sherlock-Holo** [intellij-rust/intellij-rust#5327]
## Environment

* **IntelliJ Rust plugin version:** 0.2.123.3059-201-nightly
* **Rust toolchain version:** rustc 1.45.0-nightly (65b448273 2020-05-03)
* **IDE name and version:** clion 2020.1.1
* **Operating system:** Arch Linux

## Problem description
replace all occurrences doesn't work

## Steps to reproduce

1. define a function like `get_data() -> u64`
2. use it in line 2
3. in line 1, type `get_data().let`
4. use the **replace all 2 occurrences**
5. it should create a variable at line 1 and replace line 2 `get_data()` with the variable, but not

**Comment by Sherlock-Holo** [comment]
with plugin version `0.2.125.3154-201-nightly` it still doesn't work, is there any fixing plan for this? 😢

**Comment by Kobzol** [comment]
Could you please post a specific Rust code snippet and tell us what action/refactoring are you trying to run? I don't understand what are you trying to achieve from your description. Thanks.

**Comment by Sherlock-Holo** [comment]
original code
```rust
fn main() {
    let v = Vec::new();

    use_len(len(v))
}

fn len(v: Vec<i32>) -> usize {
    v.len()
}

fn use_len(len: usize) {}
```

insert `len(v)` before `use_len(len(v))` and type `.let` to create a variable for `len(v)`, it will display 
![image](https://user-images.githubusercontent.com/10096425/84590523-0319a480-ae6a-11ea-91cf-636a91f58fde.png)

choose "replace 2", but `use_len(len(v))` doesn't replace

**Comment by dima74** [comment]
Looks likes this was fixed by #8100
